src/contracts/KeysManager.contract.js

```diff
@@ -0,0 +1,18 @@
+import Web3 from 'web3'
+import { networkAddresses } from './addresses'
+import helpers from './helpers'
+
+export default class KeysManager {
+  async init({ web3, netId }) {
+    const { KEYS_MANAGER_ADDRESS } = networkAddresses(netId)
+    console.log('KeysManager address', KEYS_MANAGER_ADDRESS)
+    let web3_10 = new Web3(web3.currentProvider)
```

Comment:
Why `let`? Could this object change somewhere in a logic? If no, let's define as `const`.

Reply:
Done in https://github.com/poanetwork/poa-dapps-voting/pull/165/commits/b6e62529fbfbfd85ca1e0e88d1d0bf0f411b0eb8.